WordPress/Classes/ViewRelated/Stats/Period Stats/SiteStatsPeriodViewModel.swift

```diff
@@ -104,4 +110,27 @@ private extension SiteStatsPeriodViewModel {
         return dataRows
     }
 
+    func searchTermsTableRows() -> [ImmuTableRow] {
+        var tableRows = [ImmuTableRow]()
+        tableRows.append(CellHeaderRow(title: PeriodHeaders.searchTerms))
+        tableRows.append(TopTotalsPeriodStatsRow(itemSubtitle: SearchTerms.itemSubtitle,
+                                                 dataSubtitle: SearchTerms.dataSubtitle,
+                                                 dataRows: searchTermsDataRows(),
+                                                 siteStatsPeriodDelegate: periodDelegate))
+
+        return tableRows
+    }
+
+    func searchTermsDataRows() -> [StatsTotalRowData] {
```

Comment:
Can/ should searchTermsTableRows and searchTermsDataRows be private?

Reply:
`private extension SiteStatsPeriodViewModel`

Private extensions FTW! 😄

Reply:
Ooh sorry! I missed that 😬